### DISCUSSION TITLE
Add multi-select fields to the projects

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [community/community#50333] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

Sometimes I can't just sort an item in one category and that's why I need this feature.

**Comment by stoutgz** [comment]
While Labels could somewhat be used for this, I agree that a single field that captures multiple possible values would be ideal to break it down to a category of tags essentially. For example, in one of my projects, we capture what contract number a body of work applies to. Some work might be broken up over multiple contracts. I'd rather not add every new contract number to my labels and just capture this as a "multi-select" where if work gets added for a given feature under more than one contract, I can add the new contract as a value to the multi-select field and I can keep the old contract number selected and add the new contract number as well.

**Comment by rileybroughten** [comment, marked as answer]
There is [another open request](https://github.com/orgs/community/discussions/6580) for a multi-select field type in Projects, and we will be providing updates there! Stay tuned as this is on our backlog 👀
